kwok/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Start from a base image
+FROM registry.access.redhat.com/ubi8:latest
+
+# Update packages
+RUN yum update -y && yum clean all
+
+RUN yum install -y bash
+
+# Install curl and other dependencies
+RUN yum install -y curl tar && yum clean all
+
+# Install Kwok & Kwokctl
+RUN KWOK_VERSION=$(curl -s https://api.github.com/repos/kubernetes-sigs/kwok/releases/latest | grep 'tag_name' | cut -d\" -f4) && \
+    curl -LO https://github.com/kubernetes-sigs/kwok/releases/download/${KWOK_VERSION}/kwok-linux-amd64 && \
+    chmod +x kwok-linux-amd64 && \
+    mv kwok-linux-amd64 /usr/local/bin/kwok && \
+    curl -LO https://github.com/kubernetes-sigs/kwok/releases/download/${KWOK_VERSION}/kwokctl-linux-amd64 && \
+    chmod +x kwokctl-linux-amd64 && \
+    mv kwokctl-linux-amd64 /usr/local/bin/kwokctl
+
+ARG OCP_VERSION=4.12.23
+RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OCP_VERSION}/openshift-client-linux-${OCP_VERSION}.tar.gz | tar xvz -C /usr/local/bin/
```

Comment:
Can use that same base image I've used in #10 which already contains `oc` though I'm not sure we really need it.

Reply:
Don't we require control over oc version?

Reply:
removed